data/db_demo.yaml

```diff
@@ -920,7 +920,8 @@ analysis_service:
     authentication_type: "header_token"
     _authinfo: '{"header_token": {"Authorization": "Bearer MY_TOKEN"}}'
     analysis_type: "lightcurve_fitting"
-    input_data_types: ["photometry", "redshift"]
+    input_data_types: ["photometry", "redshift", "spectra", "annotations", "classifications", "comments"]
+    optional_analysis_parameters: '{"source_name": ["nugent-sn2p", "salt3", "v19-1998bw", "mlcs2k2", "nugent-sn2n"], "fix_z": ["True", "False"]}'
```

Comment:
Is model_name better?

Reply:
`source` is the name of the keyword in SNcosmo. Changed that here to unify.

Reply:
ah gotcha.